tests/foreman/ui/test_repository.py

```diff
@@ -2035,3 +2035,286 @@ def test_positive_git_view_content(self):
 
         :CaseAutomation: notautomated
         """
+
+
+class FileRepositoryTestCase(UITestCase):
+    """Implements File Repo tests in UI"""
+
+    @stubbed
+    @tier1
+    def test_positive_create_with_name(self):
+        """Check File Repository creation
+
+        :id: 9f90c42e-a077-4f0a-b5ec-2b342ff2d9fe
+
+        :Setup:
+            1. Navigate to Content -> Products
+            2. Create a Product
+
+        :Steps:
+            1. Click "New Repository" button
+            2. Define a valid name and choose "type" file
+            3. Check repo is listed under "Repositories" product's tab
+
+        :expectedresults: File Repository is created and listed as product's
+            repository
+
+        :CaseAutomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_upload_file(self):
+        """Check arbitrary file can be uploaded to File Repository
+
+        :id: 2a9b74a4-cba8-430d-a699-fea543496466
+
+        :Setup:
+            1. Navigate to Content -> Products
+            2. Create a Product
+            3. Create a File Repository
```

Comment:
I changed this but am not sure why GH ui is not showing it here on conversation tab.